dotnet/docs/EXPERIMENTS.md

```diff
@@ -42,6 +42,7 @@ You can use the following diagnostic IDs to ignore warnings or errors for a part
 - SKEXP0030: MongoDB memory connector
 - SKEXP0031: Pinecone memory connector
 - SKEXP0032: Postgres memory connector
+- SKEXP0033: GoogleVertexAI Connector
```

Comment:
How will this relate to https://github.com/microsoft/semantic-kernel/pull/5347?

Reply:
AS soon the feature branch is updated with the latest main changes, this will be fixed.